File: Find the unique number.py
def find_uniq(arr):
    # your code here
    a = {}
    
    for i in arr:
        if(i in a):
            a[i] += 1
        else:
            a[i] = 1
    
    for x in a:
        if(a[x] == 1):
            return x

File: Find the unique number.js
function findUniq(arr) {
  // do magic
  var a = {}
  
  for(let i = 0; i < arr.length; i++){
    if(a[arr[i]]){
      a[arr[i]]++
    } else {
      a[arr[i]] = 1
    }
  }
  
  for(let x in a){
    if(a[x] == 1){
      return parseFloat(x)
    }
  }
}
